source/configuration_manual/imap_preview.rst

```diff
@@ -0,0 +1,50 @@
+.. _imap_preview:
+
+============
+IMAP PREVIEW
```

Comment:
This is definitely not configuration_manual. Possibly admin_manual?

Reply:
Moved to admin_manual